internal/stacks/stack_manager.go

```diff
@@ -721,9 +721,12 @@ func (s *StackManager) PrintStackInfo(verbose bool) error {
 
 func (s *StackManager) patchConfigAndRestartFireflyNodes(verbose bool) error {
 	for _, member := range s.Stack.Members {
+		configPayload := map[string]interface{}{
+			"preInit": false,
+		}
 		s.Log.Info(fmt.Sprintf("applying configuration changes to %s", member.ID))
 		configRecordUrl := fmt.Sprintf("http://localhost:%d/admin/api/v1/config/records/admin", member.ExposedFireflyAdminPort)
-		if err := core.RequestWithRetry("PUT", configRecordUrl, "{\"preInit\": false}", nil); err != nil && err != io.EOF {
+		if err := core.RequestWithRetry("PUT", configRecordUrl, configPayload, nil); err != nil && err != io.EOF {
```

Comment:
Sorry @nguyer - I've read the comment in the PR, but I'm struggling to see the difference.
Is it that `RequestWithRetry` doesn't set `Content-type: application/json` when you pass it a string? (rather than asking it to do the map->string serialization)

Reply:
... ok, got it now - so I think what you're saying is when `RequestWithRetry` was called with `{"preInit":false}` the REST client was sending this as the JSON `"{\"preInit\":false}"`.
Now if you pass a Map in, it instead is sending `{"preInit":false}`

Reply:
Yep. That's exactly what was happening. So the whole config value ended up getting set to this string, rather than a JSON object like we were intending.